apps/ecommerce/lambda/src/types.ts

```diff
@@ -1,5 +1,5 @@
 // lambda types
-export interface ResourceLink {
+export interface ExternalResourceLink {
```

Comment:
I realized we were colliding with the contentful app SDK's built-in `ResourceType` type definition, so I renamed this to be specific to our variation of this (includes `provider` attribute currently not encompassed)